### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by karayamanemre**
- Added the following validations:
      For the User model:
      Name must not be blank.
      PostsCounter must be an integer greater than or equal to zero.
- For the Post model:
      Title must not be blank.
      Title must not exceed 250 characters.
      CommentsCounter must be an integer greater than or equal to zero.
      LikesCounter must be an integer greater than or equal to zero.
- Added unit specs for all models' methods and validations.

### CODE REVIEW

File: db/migrate/20230117193352_change_column_in_table.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeColumnInTable < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :comments, :user_id, :post_id
```
Comment (GzimAsani) [inline db/migrate/20230117193352_change_column_in_table.rb]: - [x] You have some migration error, kindly check into this 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @karayamanemre                        :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your validations :+1: 

 ## Required Changes 

- [x] Be more specific on how a user can run your app, include how to run migrations.

- [x] Please add key Future Features on your Readme file and make sure you have every section included
image






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230117193352_change_column_in_table.rb (hunk 1 above)
Comment (karayamanemre) [reply db/migrate/20230117193352_change_column_in_table.rb]: i didn't add date from luxon. and i removed the change column files so i think it should work. thanks for the review.

**Review by cynthiainga — APPROVED** [review]
Hi @karayamanemre, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**